doc/flame/components.md

```diff
@@ -122,6 +123,36 @@ class GameOverPanel extends PositionComponent with HasGameRef<MyGame> {
 }
 ```
 
+There are two methods for adding children components to your component. First,
+you have methods `add()`, `addAll()`, and `addToParent()`, which can be used
+at any time during the game. Traditionally, children will be created and added
+from the component's `onLoad()` method, but it is also common to add new
+children during the course of the game.
+
+The second method is to use the `children:` parameter in the component's
+constructor. This approach more closely resembles the standard Flutter API:
+```dart
+class MyGame extends FlameGame {
+  @override
+  Future<void> onLoad() async {
+    add(
+      PositionComponent(
+        position: Vector2(30, 0),
+        children: [
+          HighScoreDisplay(),
+          HitPointsDisplay(),
+          FpsCounter(),
+        ],
+      ),
+    );
+  }
+}
+```
+
+The two approaches can be combined freely: the children specified within the
+constructor will be added first, and then any additional child components
+after.
```

Comment:
Shall we also clarify, if the added children via `children` are accesible (or not) from an lifecycle method such as `onLoad` by reading `children`?

Reply:
Added clarification.